tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 minversion = 3.7.0
 envlist =
-    py{27,36,37}-ansible{24,25,26,27}-{functional,unit}
-    py{27,36,37}-lint
+    py{27,35,36,37}-ansible{25,26,27}-{functional,unit}
+    py{27,35,36,37}-lint
```

Comment:
Could you please put it first? Linters are faster so it should be better feedback timing this way

Reply:
this was fixed in master